ansible/roles/ros_workspace/files/vs_code/one_line_install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright 2023 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+build_tools_dir=~/sr-build-tools
+build_tools_branch=F_#SP-476_vs_code_setup_lint
```

Comment:
Likewise. We should probably point to master before we merge.

Reply:
Addressed in https://github.com/shadow-robot/sr-build-tools/pull/697/commits/89c8670d4f1388b9f4219295f7f553aa6b8b76ed